### ISSUE TITLE
Clarifying and opening up Blinded bane

### ISSUE THREAD

**Issue by LockeVS** [openlegend/core-rules#205]
This is a branch-off from the Discord and MightyBell discussion around the proposed "Dulled Senses" bane taking the place of the current Blinded.

The primary concern is that while Blinded is easy for many players, new and old alike, to understand and relate to, the bane as-written is too specific in affecting only sight. One proposed suggestion was to change Blinded to "Dulled Senses" or "Impaired," allowing the person invoking the bane to specify which of the target's senses was affected (one Dulled Senses or Impaired bane that may affect any sense, not a new bane for each sense).

Brian has already said he's not in favor of the proposed Dulled Senses/Impaired change, but Eli mentioned that the current wording for Blinded may still be too restrictive and suggested posting this issue here as a discussion reminder. (I've included the original concept behind Dulled Senses/Impaired just for reference.)

**Comment by Th3Way** [comment]
In addition, the current reading of Blinded applies to all perception rolls, not just those relating to sight.

A rewording to "The target has a 50% chance of automatically failing any Perception checks or attack rolls that rely upon the selected sense." Covers it better. Allowing for multiple conditions to be covered by a single bane and no additional rules needed.

It was also mentioned that 50% rule will be adjusted to advantage. I'm concerned this will result in the penalty being extremely minor and not nearly as much of a hindrance. I am of the mind that this 50% rule is clunky and not in keeping with all other dice methodology the system has, however, it does handle the extreme penalty in a very simple to understand way.

**Comment by DukeAnax** [comment]
@Th3Way the bane clearly states that you can not see, that means you actually fail 100% on perseption checks that rely on sight, as it should be, you are blind. 

The 50%/disadvantage applies to _general_ perseption checks where you have the other senses to help out.

If disadvantage is not enough it can be raised to disadvantage 2 or3. Which menas you are very likely to roll 1d20+1 or 3d20 take low.

**Comment by ucffool** [comment]
I'd definitely be up for changing the "50%" because the system has a built-in ADV/DIS system that should be used. Disadvantage one is brutal IF you have a 0 in your Perception... it should be. If you have even just 1 point in that attribute (or the enemy does), then it results in a reduction of 1.5/20 on the roll average, so not brutal.

**Comment by DukeAnax** [comment]
I agree that there needs to be something for at least deafened and balance. however smell, touch and maybe even taste may be important for some non humans, and some beasts may have even stranger senses. making it all one bane would be the simplest solution, how about this:

# Senseless
**Duration**: Resist ends (Fail x 3 = 1 minute)

**Power Level**: 4

**Attack Attributes**: Agility, Creation, Energy, Entropy, Influence

**Attack**:

Agility vs. Evasion
Creation vs. Evasion
Energy vs. Evasion
Entropy vs. Toughness
Influence vs. Resolve

**Description:**
You obstruct your opponents senses with anything from a fistfull of sand to the eyes, a loud bang to the ears or a cloud of gas to the nose.

**Effect:**
Choose any senses that are affected by your attack, the target can not use those senses as long as the effect persists. Any actions that make use of those senses receive disadvantage on their roll. If the target depends heavily on one of the senses the GM may impose additional disadvantage.


I first tried to quantify that last bit more, but it was really complex, then I realized: this is Open Legends and I don't have to. Leaving it open to the players and the GM like this works just fine.

**Comment by brianfeister** [comment]
FYI, we're duplicating conversation between Mightybell and Github on this one.

I'm not a fan of "Dulled Senses" / "Senseless" / similar overloaded terms because new players will not think to look for the very normal / expected "Blinded" bane by a different name. 

Yes, Robert O'Neill and I are discussing a different way of handling the penalty with Disadvantage instead of 50% miss.

And we will likely create a "Deafened" bane and ignore the other senses as GM House Rule

**Comment by brianfeister** [comment]
And we will clarify that it's `Perception rolls involving sight`

**Comment by DukeAnax** [comment]
@brianfeister are you aware that the CONTRIBUTING.md on this Github basically instructs us to duplicate all conversation between Mightybell and Github?

**Comment by brianfeister** [comment]
That's not true at all @DukeAnax. It says discuss first on MightyBell and come up with an actionable solution and then propose it here on Github.

I'm not in anyway upset with @LockeVS or anyone on this thread, but if we were following `CONTRIBUTING.md` as law, we would have waited until a specific proposal was formulated from the discussion and then opened that as a Github issue.

Again, I'm not bothered. I'm mainly explaining that people on this thread are missing the comments I've already made on the other thread. Had the policy of waiting until something specific was formulated been followed, we wouldn't have had to repeat ourselves.

**Comment by DukeAnax** [comment]
Oh, I missed part of that discussion, particular your posts shutting it down, so I though that up here was the actionable solution.
